src/Adapters/MockAdapter.ts

```diff
@@ -680,6 +681,27 @@ export default class MockAdapter
         }
     }
 
+    async advancedSearchADTTwins(
+        _params: AdapterMethodParamsForAdvancedSearchADTwins
+    ) {
+        try {
+            await this.mockNetwork();
+
+            return new AdapterResult({
+                result: new ADTAdapterTwinsData({
+                    // TODO: filter out based on properties from params
```

Comment:
it'd be great if we could do the filtering here so we get actual results in the story.

Reply:
filtered if user selects equals